cloud/aci-integration.md

```diff
@@ -106,6 +106,13 @@ $ docker context use myacicontext
 $ docker run -p 80:80 nginx
 ```
 
+> **Note**
+>
+> Unlike the Azure portal - which [does not require](https://stackoverflow.com/a/57454806) the full URI of DockerHub images - the Docker Azure Integration seems to default to Azure Container Registry in order to determine the source registry from where to pull the specified image(s). Therefore you may need to add the full URI of the image(s) if you are pulling from anywhere other than azurecr.io. In case of images hosted on DockerHub, you may therefore specify:
```

Comment:
@syedsuhaib Thanks for the PR. Suggest we remove links to third-party sites from the Docs. ;)

Reply:
I'm sorry, maybe I'm missing something, but I can run an image that I just pushed on Docker hub (I pushed some nginx under my hub account) : 
```
$  docker --context aci run gtardif/nginx-test
[+] Running 2/2
 ⠿ Group gracious-kare  Created                       4.8s
 ⠿ gracious-kare        Created                            20.8s
/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
/docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
/docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
/docker-entrypoint.sh: Configuration complete; ready for start up
2022/07/21 16:51:20 [notice] 19#19: using the "epoll" event method
2022/07/21 16:51:20 [notice] 19#19: nginx/1.23.1
2022/07/21 16:51:20 [notice] 19#19: built by gcc 10.2.1 20210110 (Debian 10.2.1-6) 
2022/07/21 16:51:20 [notice] 19#19: OS: Linux 5.10.102.2-microsoft-standard
2022/07/21 16:51:20 [notice] 19#19: getrlimit(RLIMIT_NOFILE): 1024:1048576
2022/07/21 16:51:20 [notice] 19#19: start worker processes
2022/07/21 16:51:20 [notice] 19#19: start worker process 48
```

Reply:
No, it doesn't work for me either. and I don't understand to where I am supposed to get the URI from.

Reply:
> @syedsuhaib Thanks for the PR. Suggest we remove links to third-party sites from the Docs. ;)

I've removed the external link but the PR is now closed. I should perhaps open a new one now?